### PR TITLE
Add heroku ci:open to show CI on Dashbaord from CLI

### LINKED ISSUE #21: Feature request: open dashboard to test screen from CLI
Something like:

```
heroku ci:open <<optional run number>>
```

and that opens to the pipelines test page.

### PR DESCRIPTION
**Pull request by maxbeizer**
closes #21 

@heroku/devex I took a crack at this late on Friday. I'm not wild about adding another dependency, but I wasn't sure about OS compatibility issues. Perhaps, I'll ask the CLI team. Mostly, I struggled with designing a test for this. If you all have ideas, I'd love to hear them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "got": "^6.6.3",
     "heroku-cli-util": "^6.0.15",
     "heroku-run": "^3.4.3",
+    "opn": "^4.0.2",
```
Comment (raulb) [inline package.json]: @maxbeizer I don't think it's necessary including a new package for this as this is something we can do using `heroku-cli-util` with `cli.open`. For example: https://github.com/heroku/heroku-orgs/blob/a84ad8395017177452678182cfac445d7fa20844/commands/orgs/open.js.

Comment (maxbeizer) [inline package.json]: Nice! I didn't know that was in there. I'll update this PR. Thanks @raulb

**Comment by raulb** [comment]
@maxbeizer thanks!

**Comment by raulb** [comment]
```
› np

Publish a new version of heroku-ci (1.4.2)

? Select semver increment or specify new version minor 	1.5.0
? Will bump from 1.4.2 to 1.5.0. Continue? Yes
 ✔ Prerequisite check
 ✔ Git
 ✔ Cleanup
 ✔ Installing dependencies using npm
 ✔ Running tests
 ✔ Bumping version
 ✔ Publishing package
 ✔ Pushing tags

 heroku-ci 1.5.0 published 🎉
```

**Comment by gudmundur** [comment]
Sweet! Nice work @maxbeizer.

**Pull request merged**